config.pri

```diff
@@ -14,11 +14,14 @@ QT_VER_MAJ = $$member(QT_VERSION, 0)
 QT_VER_MIN = $$member(QT_VERSION, 1)
 
 lessThan(QT_VER_MAJ, 5) {
-error(YACReader requires Qt 5 or newer but Qt $$[QT_VERSION] was detected.)
-  }
+  error ("You need at least Qt 5.6 to compile YACReader or YACReaderLibrary.")
+}
+
+equals(QT_VER_MAJ, 5) {
 lessThan(QT_VER_MIN, 6):!CONFIG(no_opengl) {
```

Comment:
This `:!CONFIG(no_opengl)` subcondition suggests that YACReader can be built with Qt5 version < 5.6 if `no_opengl` config variable is set. If so, then the error message replacement in the `lessThan(QT_VER_MAJ, 5)` block is misleading and the old error message should probably be restored. If Qt < 5.6 is not supported at all, perhaps `:!CONFIG(no_opengl)` should be removed.

Reply:
I think this was added for some users stuck on older versions of Qt on Debian or Mepis Linux. The only part of YACReader requiring Qt 5.6 at that time was the cover flow, so you could still build with older versions by disabling the opengl widgets. That likely is no longer the case.

I think we should use the occasion and increase the minimum version requirements for Qt. There are probably not many people left using these ancient Qt versions to build YACReader and the LTS versions of Debian and Ubuntu that are the main target for such legacy builds already have newer versions packaged. Given that our current code base uses functionality from Qt 5.8 and Ubuntu 18.04 ships with Qt 5.9 I suggest using that version as our new minimum target.